pkg/iso3166/alpha_test.go

```diff
@@ -58,3 +61,8 @@ func TestNormalizedSearch(t *testing.T) {
 		}
 	}
 }
+
+func TestList(t *testing.T) {
+	codes := iso3166.List()
+	require.Equal(t, len(codes), 249)
```

Comment:
Small thing but I usually put the expected value first and actual value second to match the convention defined in `require.Equal`.